gf-ui-lib/src/components/Button/Button.tsx

```diff
@@ -14,21 +15,32 @@ interface ButtonProps {
   onClick: () => void;
   /**
    * Button type
+   * If type is undefined, then the button type will be
    */
-  type: "submit" | "button";
+  type?: "submit" | "button";
+  /**
+   * Button size.
+   * If size is undefined, then the button will have parentWidth style
+   */
+  size?: "innerContent" | "parentWidth";
 }
 
 /**
  * Button component
  */
 export const Button: React.FC<ButtonProps> = (props: ButtonProps) => {
+  const buttonClassNames = `${styles.button} ${props.size && styles[props.size]}`;
```

Comment:
Use typescript to define props types component, don't forget about default props (props.size can't be 0, '', null or undefined, don't check it

Reply:
add default states to type and size